resource_processor/shared/config.py

```diff
@@ -45,7 +45,7 @@ def get_config(logger_adapter) -> dict:
     config["porter_env"] = {
         "HOME": os.environ["HOME"],
         "PATH": os.environ["PATH"],
-        "KEY_VAULT_NAME": config["key_vault_name"],
+        "KEY_VAULT_URL": config["key_vault_url"],
```

Comment:
I'm not 100% we should change this one.
Where does porter use this environment variable?